examples/Rules-method-nextBest-NextBestProbMTDLTE.R

```diff
@@ -0,0 +1,41 @@
+# Example of usage for `NextBestProbMTDLTE` NextBest class.
```

Comment:
```suggestion

# Example of usage for `NextBestProbMTDLTE` NextBest class.
```
one empty line before

Reply:
done

Reply:
@wwojciech @danielinteractive I did add an empty line at the beginning of any example, but [[skip actions] Restyle files](https://github.com/Roche/crmPack/pull/484/commits/bc30398ba1c13fef4f7b8a0ccd99742014d6df8f) seems to make automatic changes to pushed files. One of the changes is to remove empty lines at the beginning of a file.